### DISCUSSION TITLE
Shared UI Components across platform

### DISCUSSION THREAD

**Discussion by zdmc23** [DiscipleTools/disciple-tools-theme#1103] [Ideas and Requests]
I propose leveraging Expo or Ionic to develop Shared UI components (Custom Field Types, Tiles, Custom Post Types), which would consolidate testing efforts, and would also ease any transition toward PWA or a "headless" D.T. (These components would be in React, so we will be able to reuse existing code).

Ref: https://github.com/DiscipleTools/disciple-tools-mobile-app/discussions/540

**Comment by mikeallbutt** [comment]
I support this idea if it will make DT be more easily developed and used by more people.
